docs/source/requests-new.md

```diff
@@ -1386,6 +1388,62 @@ Command to change Pool's configuration
 }
 ```
 
+### LEDGERS_FREEZE
+
+Disable specific legders. System completely ignore frozen ledgers. After freeze you can delete frozen ledger via [remove_ledger.py] (https://github.com/hyperledger/indy-node/blob/master/scripts/remove_ledger.py).
```

Comment:
I'm not sure that "disable" is a good word here. I suggest to use "Freeze deprecated ledgers"
Could you add more details about static and dynamic validations for this request.

Reply:
And it's not completely ignored in fact, because we still write its hashes to audit ledger.